content/integrations/blog/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Blog"
+date: 2018-07-30T18:34:00-07:00
+draft: false
+class: "resized-logo"
+---
+
+![OpenCensus Logo](https://opencensus.io/img/logo-sm.svg)
+
+{{% children %}}
```

Comment:
Oh wait, I believe we meant to put this under `content/blogs/` so
```
content
   |- blogs
       |- _index.md
       |- iot.md
```

because right now it is under `content/integrations/blog`
<img width="959" alt="screen shot 2018-08-09 at 3 31 47 pm" src="https://user-images.githubusercontent.com/4898263/43929374-978a2f30-9be9-11e8-85e2-103b613128ce.png">

Perhaps a follow up PR to reorganize it before we share links?

Reply:
okay sure

Reply:
Looks great otherwise, great work @hbgxsm95!

Reply:
Oh that's correct. Please change the directory name to plural: "blog" -> "blogs".

Reply:
@hbgxsm95 I can send one and you can help me review please?